examples/canal_demo.py

```diff
@@ -0,0 +1,50 @@
+"""
+This demo shows a usage of an Lawicel CANUSB device.
+"""
+
+import sys, os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+import can
+
+# import logging
+# logging.basicConfig(level=logging.DEBUG)
+
+serialMatcher="PID_6001"
+bus = can.interface.Bus(bustype="canal", dll="canusbdrv64.dll", serialMatcher=serialMatcher, bitrate=100000, flags=4)
+
+# alternative: specify device serial:
+# bus = can.interface.Bus(bustype="canal", dll="canusbdrv64.dll", serial = "LW19ZSBR", bitrate=100000, flags=4)
+
+def send_one():
+    msg = can.Message(arbitration_id=0x00c0ffee,
+                      data=[0, 25, 0, 1, 3, 1, 4, 1],
+                      extended_id=True)
+    try:
+        bus.send(msg)
+        print ("Message sent:")
```

Comment:
I suggest adding a `from future import print` and removing the space after `print` here